unit_testing/test_labels.py

```diff
@@ -216,3 +233,17 @@ def test_remove_other_labels_from_image(test_image):
     diff = res.data == expected.data
 
     assert diff.all()
+
+
+@pytest.mark.parametrize("test_image", test_images)
+def test_find_missing_label(test_image):
+    img = test_image.copy()
+    false_positive = test_image.copy()
+
+    # modifying the data to create one false negative and one false positive
+    # label 1500 is not in test_image originally, and label 111 is.
+    false_positive.data = np.where(false_positive.data == 111, 1500, false_positive.data)
+    FP, FN = sct_labels.find_missing_label(false_positive, img)
```

Comment:
I notice that this looks a little different than the `test_compute_mean_squared_error_warning` test. Is there a reason why 2 different ways were used? Might be good to be consistent here.

Reply:
The reason was: this way is needed for `test_image` but not for `fake_3dimage_sct`. I will make it more consistant.

Reply:
I made it more consistent across test